modules/nw-udn-cr.adoc

```diff
@@ -0,0 +1,96 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-udn-cr_{context}"]
+= Creating a UserDefinedNetwork custom resource
+
+The following procedure sets up a user-defined network that is namespace scoped.
+
+.Prerequisites
+* You must
+.Procedure
+
+[NOTE]
+====
+If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
+====
+
+. Create a request for a user-defined network:
+
+.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content like the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-1 # <1>
+  namespace: default
+spec:
+  topology: Layer2 # <2>
+  layer2:
+    role: Primary # <3>
+    mtu: 9000 # <4>
+    subnets:
+      - "10.0.0.0/24"
+      - "2001:db8::/64" # <5>
+    excludeSubnets:
+      - "10.0.0.0/26"
+      - "2001:db8::/80" # <6>
+    JoinSubnets: [] # <7>
+    IPAMLifecycle: Persistent # <8>
+----
+<1> Name of your `UserDefinedNetwork` resource. This should not be `default`.
+<2> The layer you want to create the network on. The `topology` field accepts values of `Layer2` and `Layer3`.
+<3> Specifies `Primary` or `Secondary`. `Primary` is the only `role` specification supported in {product-version}.
+<4> The maximum transmission units (MTU) the default value is `1400`.
+<5> Specifies the subnet to be used for the network across the cluster. Supports both IPv6 and dual-stack. For example, `192.168.100.0/24`,`2001:DBB::/64`. Dual-stack may set two subnets otherwise only one is allowed. When the `topology` field is set to `Layer3`, the subnet is split into smaller subnets for every node. Accepted format for subnets when `Layer3` is set are: `10.128.0.0/16/24`. For `Layer2` values in `topology` field, standard CIDR ranges are accepted. If omitted the network only provides `Layer2` communication and you must configure IP addresses.
```

Comment:
Better to not use ``10.128.0.0/16/24`.` as example due to  it's default network should not be used for udn pods.

Reply:
fixed